.gitlab-ci.yml

```diff
@@ -11,19 +11,25 @@ stages:
   - post
 
 .shared:
-  install-dependencies:
-    - echo 'Installing Unity Hub...'
+  init-scripts:
+    # Running with no arguments will ensure our Python venv is initialized
+    - ./run-script
```

Comment:
minor: this name is very generic. By looking inside it I see that it runs python script, so I would suggest renaming this file to be closer to its purpose/content.